netlogo-gui/src/main/lab/Worker.scala

```diff
@@ -119,14 +149,14 @@ class Worker(val protocol: LabProtocol)
             ws.world.mainRNG.setSeed(value.asInstanceOf[java.lang.Double].longValue)
         }
```

Comment:
I used this approach and its close to no code written. but I've not yet pushed the code(waiting for an  answer to re-basing).